packages/@o3r/design/src/core/design-token/parsers/design-token.parser.ts

```diff
@@ -0,0 +1,172 @@
+import { promises as fs } from 'node:fs';
+import type { DesignTokenVariableSet, DesignTokenVariableStructure } from './design-token-parser.interface';
+import type {
+  DesignToken,
+  DesignTokenExtensions,
+  DesignTokenGroup,
+  DesignTokenGroupExtensions,
+  DesignTokenNode,
+  DesignTokenSpecification
+} from '../design-token-specification.interface';
+import {
+  DesignTokenTypeStrokeStyleValue,
+  isDesignToken,
+  isDesignTokenGroup,
+  isTokenTypeStrokeStyleValueComplex
+} from '../design-token-specification.interface';
+
+interface ParentReference {
```

Comment:
Why not using the one from `packages/@o3r/design/src/core/design-token/parsers/design-token-parser.interface.ts` (not exposed yet)